projects/docs/docs/manifest/api-reference.md

```diff
@@ -0,0 +1,21 @@
+---
+title: ProjectDescription.framework
+slug: '/manifest/api-reference'
+description: 'This page collects manifest API references.'
+---
+
+The API documentation for releases and main are available here:
+
+* [latest (3.0.0)](https://tuist.github.io/tuist/3.0.0/documentation/projectdescription/)
```

Comment:
Possibly, yes. Though my frontend skills are a bit out-dated :') I am trying to find different solutions (that's why I left the PR in Draft)
Potentially, we could change the workflow script, so that a special page (latest) is created that points to the latest version. That would solve the "point to the latest", though, I would like to remove the need of having to list the other versions manually either...
I need to give some other thoughts, any idea is welcomed!

Reply:
I think having the latest and explaining how to get all the other is enough